velodyne_pointcloud/src/lib/rawdata.cc

```diff
@@ -327,7 +329,9 @@ inline float SQR(float val) { return val*val; }
     float x, y, z;
     float intensity;
 
-    const raw_packet_t *raw = (const raw_packet_t *) &pkt.data[0];
+    const raw_packet_vlp16_t *raw = (const raw_packet_vlp16_t *) &pkt.data[0];
```

Comment:
Go ahead and change the VLP-16-specific stuff to be sensor-generic and I'll test the changes on a VLP-32C and an HDL-64E.

Reply:
Would it be possible for you to send me a PCAP of each? I made the raw_packet_vlp16_t to access the factory bytes (the packet definition is different from the rest of the lidars) I could technically just follow the datasheets to modify the other `unpack` function but it will likely be wrong without a pcap for me to test.

Reply:
Thanks, Andre. I'm working on getting you PCAPs for the two sensors I mentioned.